modules/update-preparing-forced-update-risk.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_content-type: CONCEPT
+[id="update-preparing-forced-update-risk"]
+= Assessing the risk of forced updates
+
+If there is no other way to update a cluster, you can "force" an update by using the `--force` option in the CLI, which allows you to forcefully update the cluster even when the update release image validation fails and the cluster is reporting errors.
+
+[IMPORTANT]
+====
+Use of --force is not recommended for updates. Please read the whole section to understand the risks of using force for updates.
+====
+
+The following text will be returned by using the `$ oc adm upgrade --help` command in the CLI:
```

Comment:
As a whole/from a content perspective, I think that this reads weird. First we're talking about using force, and then we go into using the `$ oc adm upgrade --help` command. I know that there's a relationship here by filling in some missing gaps, but I wonder if relationship between the necessity of this command + forced upgrading could be made clearer to make the content flow better (note that this is just me being nitpicky). 

I almost think it would be best organized as follow, though take it or leave it as you will. I think even this could be slightly revised to remove content (namely the two sentences about the CVO, because I think the same message gets across if you cut that part out and just say "Bypassing verification failure. . ."), but this way keeps all of the content that you already wrote. 

```suggestion
When updating a cluster, it's essential to understand the available options and their implications. For detailed information on cluster updates and available options, you can refer to the command-line documentation using the following command:

[source,terminal]
----
$ oc adm upgrade --help
----

When running this command, the following information is returned:

[source,terminal]
----
`Cluster-side guards include checks for release verification and upgradeable conditions. Again, it is usually best to give these conditions time to resolve, or to actively work to resolve them. But if you decide to trigger the update regardless of these concerns, use --force, which is passed through to ClusterVersion’s spec.desiredUpdate.force.`
----

If you find that you cannot resolve the conditions that are blocking your update, you can _force_ an update by using the `--force` option in the CLI. This allows you to forcefully update the cluster even when the update release image validation fails and the cluster is reporting errors.

The Cluster Version Operator (CVO) runs precondition checks before starting an update to determine if the cluster has the `Upgradeable=false` status condition, or any other warning condition. For example, the CVO verifies the target release signature and if the verification fails, the CVO returns `ReleaseAccepted=False`. Bypassing verification failure and forcing an update comes with potential risks, including downtime or data loss. 

Use caution when upgrading a cluster with the `--force` option.

```